frontend/src/store/event.js

```diff
@@ -89,6 +90,17 @@ const event = {
     async [actions.WITHDRAW_QUESTION]({ state }, questionId) {
       await axios.delete(`/events/${state.eventId}/questions/${questionId}`)
     },
+    async [actions.HIGHLIGHT_QUESTION]({ state, commit }, { questionId, action }) {
```

Comment:
We need to unify this with the vote action by another action called `patchQuestion`